.github/workflows/copr_build.yml

```diff
@@ -0,0 +1,92 @@
+name: copr
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened]
+env:
+  COPR_ACCOUNT: '@sssd'
+  COPR_PROJECT: pr${{ github.event.pull_request.number }}
+jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    outputs:
+      srpm: ${{ steps.srpm.outputs.file }}
+      chroots_json: ${{ steps.chroots.outputs.json }}
+    permissions:
+      contents: read
+    steps:
```

Comment:
Could you add a newline between steps for readability? 

I would also like to have a comment added similar to the commit message which describes the high level steps. IMO it is not easy to discern this just from quickly reading this .yml file

Reply:
Sure, done.